sources/packages/backend/libs/integrations/src/institution-integration/ece-integration/models/ece-integration.model.ts

```diff
@@ -36,3 +36,17 @@ export enum RecordTypeCodes {
   ECEDetail = "2",
   ECETrailer = "3",
 }
+
+export const AWARD_DETAILS_KEY = "awardDetails";
```

Comment:
Is there any reason to not use the `awardDetails` as below?
```ts
export interface ECEDisbursements {
  [disbursementScheduleId: string]: {
    institutionCode: string;
    applicationNumber: string;
    awardDetails: {
      payToSchoolAmount: number;
      isEnrolmentConfirmed: boolean;
      enrolmentConfirmationDate: Date;
    }[];
  };
}
```
The other areas can just use it, for instance
![image](https://github.com/bcgov/SIMS/assets/61259237/81362cae-d8d9-4581-96d5-93d187814401)